ivy_tests/test_ivy/test_frontends/test_tensorflow/test_mathematical_functions.py

```diff
@@ -0,0 +1,39 @@
+# global
+import numpy as np
+from hypothesis import given, strategies as st
+
+# local
+import ivy
+import ivy_tests.test_ivy.helpers as helpers
+
+
+valid_float_dtypes = (ivy.float16, ivy.float32, ivy.float64)
```

Comment:
again, we shouldn't need any local `dtype` tuples, the helper function will skip all invalid dtypes automatically.